Deployment/azuredeploy.json

```diff
@@ -114,6 +114,27 @@
         "description": "The pricing tier for the QnAMaker service."
       }
     },
+    "defaultCulture": {
+      "type": "string",
+      "allowedValues": [
+        "en",
+        "ar",
+        "de",
+        "es",
+        "fr",
+        "he",
+        "ja",
+        "ko",
+        "pt",
```

Comment:
pt is usually different between pt-pt and pt-br, so that needs to be split.

Reply:
Updated it to pt-BR as we initially got translation for it only.